src/main/java/com/iota/iri/service/API.java

```diff
@@ -1027,49 +1378,96 @@ private AbstractResponse getBalancesStatement(final List<String> addresses, fina
-      * You need to supply <code>branchTransaction</code> as well as <code>trunkTransaction</code> (the tips which you're going to validate and reference with this transaction) - both of which you'll get through the <code>getTransactionsToApprove</code> API call.
-      *
-      * The returned value is a different set of tryte values which you can input into <code>broadcastTransactions</code> and <code>storeTransactions</code>.
-      * The last 243 trytes of the return value consist of the: <code>trunkTransaction</code> + <code>branchTransaction</code> + <code>nonce</code>.
-      * These are valid trytes which are then accepted by the network.
-      *
-      * @param trunkTransaction the trunk transaction
-      * @param branchTransaction the branch transaction
-      * @param minWeightMagnitude the minimum weight magnitute
-      * @param trytes the list of trytes to attach
-      * @return trytes the list of transactions in trytes
+      * <p>
+      *     Attaches the specified transactions (trytes) to the Tangle by doing Proof of Work.
+      *     You need to supply <code>branchTransaction</code> as well as <code>trunkTransaction</code>.
+      *     These are the tips which you're going to validate and reference with this transaction. 
+      *     These are obtainable by calling the <code>getTransactionsToApprove</code> API call.
+      * </p>
+      * <p>
+      *     The returned value is a different set of tryte values which you can input into 
+      *     <code>broadcastTransactions</code> and <code>storeTransactions</code>.
+      *     The last 243 trytes of the return value consist of the following:
+      *     <ul>
+      *         <li><code>trunkTransaction</code></li>
+      *         <li><code>branchTransaction</code></li>
+      *         <li><code>nonce</code></li>
+      *     </ul>
+      *     These are valid trytes which are then accepted by the network.
+      * </p>
+      * @param trunkTransaction A reference to an external transaction (tip) used as trunk.
+      *                         The transaction with index 0 will have this tip in its trunk.
+      *                         All other transactions reference the previous transaction in the bundle (Their index-1).
+      *                         
+      * @param branchTransaction A reference to an external transaction (tip) used as branch.
+      *                          Each Transaction in the bundle will have this tip as their branch, except the last.
+      *                          The last one will have the branch in its trunk.
+      * @param minWeightMagnitude The amount of work we should to to confirm this transaction. 
+      *                           Each 0-trit on the end of the transaction represents 1 magnitude. 
+      *                           A 9-tryte represents 3 magnitudes, since a 9 is represented by 3 0-trits.
+      *                           Transactions with a different minWeightMagnitude are compatible.
```

Comment:
```suggestion
      *                           Transactions with a higher minWeightMagnitude are compatible.
```
?

Reply:
@alon-e Some people were confused as to use of 14 or 15, or anything else.
and if you make all your transactions MWM 20, it is still compatible with lower, so not specifically higher only